### PR TITLE
Fix Newtonsoft.Json.JsonReaderException

### PR DESCRIPTION
**Pull request by mikdav**
Fix Issue #17 Newtonsoft.Json.JsonReaderException when a device has more than 2GB of total memory.

### CODE REVIEW

File: UniFiSharp/NetworkDeviceConfigurations.cs [hunk 1]
```diff
@@ -32,21 +36,17 @@ public abstract class NetworkDeviceConfigurations
...
 
-        public static Func<bool, NetworkDeviceConfigurations<bool>> SwitchVlanEnabled =>
-            new Func<bool, NetworkDeviceConfigurations<bool>>((v) =>
-                new NetworkDeviceConfigurations<bool>((object)new { switch_vlan_enabled = v }));
-
         public static Func<string, NetworkDeviceConfigurations<string>> LedOverride =>
             new Func<string, NetworkDeviceConfigurations<string>>((v) =>
                 new NetworkDeviceConfigurations<string>((object)new { led_override = v }));
 
-        public static Func<string, NetworkDeviceConfigurations<string>> Name =>
```
Comment (anthturner) [inline UniFiSharp/NetworkDeviceConfigurations.cs:43]: Was this intentional? Not sure I'm following the purpose. Seems like things are just being shuffled around in this file.

**Comment by anthturner** [comment]
Looks great -- I left a minor comment, then we can merge. 👍

**Comment by mikdav** [comment]
@anthturner, I don't see a comment anywhere in the PR, the files, or the issue.  Can you tell me where you left it?

**Comment by anthturner** [comment]
If you look here:

https://github.com/anthturner/UniFiSharp/pull/21/files/86ee4b4ca83d215e60975a77c837f5dfa9ab8365#diff-13167a673a54979905bb9e64cd26af954c6a5d6c19f4c5300eefbb4e46f51f3e

On line 43, it shows the comment inline. It's part of the "review" process for PRs on GitHub; it allows you to respond directly to the specific concern with your own comments too. :)

Incidentally, if you scroll up in this PR it should show as well, next to
> **anthturner** started a review

**Review by anthturner — APPROVED** [review]
I've decided this isn't important enough to block integration.

**Comment by mikdav** [comment]
Ha.  That comment never showed up on the file in my view of the pull request until now.  Sorry, I was going to follow up with a screenshot of no comments on the pull request, but then got distracted by other work.

In answer to your question—yes it was intentional.  A bit of OCD.  Putting the NetworkConfigurations in alphabetical order for readability.  I just have some standard improvements I do in any file I touch.  But its your project.  If changes like that bother you, I’ll stick to the minimum necessary in the future.

**Comment by anthturner** [comment]
Not a problem; I merged it anyways. 😀

Thanks for your contribution!

**Pull request merged**